src/control-plane/backend/lambda/api/service/stack.ts

```diff
@@ -304,11 +305,16 @@ export class StackManager {
   }
 
   private async getExecutionDetail(executionArn: string): Promise<DescribeExecutionOutput | undefined> {
-    const params: DescribeExecutionCommand = new DescribeExecutionCommand({
-      executionArn: executionArn,
-    });
-    const result: DescribeExecutionCommandOutput = await sfnClient.send(params);
-    return result;
+    try {
+      const params: DescribeExecutionCommand = new DescribeExecutionCommand({
+        executionArn: executionArn,
+      });
+      const result: DescribeExecutionCommandOutput = await sfnClient.send(params);
+      return result;
+    } catch (error) {
+      logger.error('get execution detail error ', { error });
```

Comment:
Let's handle the known case of execution that does not exist.

https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/client/sfn/command/DescribeExecutionCommand/

Also, I prefer to using `info` for known case and `warn` for logging other cases.